dev/INSTALL

```diff
@@ -77,7 +81,7 @@ General Instructions for Deployment of a New Release
 
  12. Start the datatracker service (it is no longer necessary to restart apache) ::
 
-      exit # or CTRL-D, back to root level shell
+      exit 
       sudo systemctl start datatracker.service datatracker.socket
 
  13. Start async task worker and message broker:
```

Comment:
Did you want to call out the switch back to the docker-group user?

Reply:
I'll leave it as is for now.